model-optimizer/extensions/front/onnx/activation_ext.py

```diff
@@ -38,6 +38,16 @@ def extract(cls, node):
         return cls.enabled
 
 
+class AcoshExtractor(FrontExtractorOp):
```

Comment:
How about Tensorflow: https://www.tensorflow.org/api_docs/python/tf/math/acosh  and other frameworks?